{{cookiecutter.project_slug}}/setup.cfg

```diff
@@ -32,14 +32,12 @@ include_package_data = True
 packages = find:
 zip_safe = False
 
-[options.packages.find]
-exclude = tests, tests.*
-
 [bdist_wheel]
 universal = 1
 
 [tool:pytest]
 norecursedirs = .* build dist venv test_data
+DJANGO_SETTINGS_MODULE = {{cookiecutter.project_name}}.tests.settings
```

Comment:
This should use `project_slug` instead. (Noticed while was testing this change.)